docs/getting-started/self-hosting.mdx

```diff
@@ -272,12 +272,50 @@ Our TestDriver Golden Image (AMI) comes pre-configured with everything you need
 
 ### Modifying the AMI
 
-You can customize the AMI for your specific needs:
+<Danger>
+  **Change Default Password First**: The TestDriver AMI includes a default password for initial access. You MUST change this password before saving a new AMI. Never use the default password in production environments.
+</Danger>
+
+You can customize the AMI for your specific needs. Follow these steps carefully:
+
+#### Step 1: Initial Access
+
+Connect to your instance via RDP using the default credentials:
```

Comment:
What's the best way to do this?  Ideally via the AWS Console securely, but they have to get around this error:

<img width="3332" height="624" alt="Image" src="https://github.com/user-attachments/assets/ae0d5dca-a829-482e-8b7f-379e32d77263" />